addons/l10n_ro_efactura/wizard/account_move_send.py

```diff
@@ -0,0 +1,82 @@
+from odoo import api, fields, models, _
+
+
+class AccountMoveSend(models.TransientModel):
+    _inherit = 'account.move.send'
+
+    l10n_ro_edi_send_enable = fields.Boolean(compute='_compute_l10n_ro_edi_send_enable')
+    l10n_ro_edi_send_readonly = fields.Boolean(compute='_compute_l10n_ro_edi_send_readonly')
+    l10n_ro_edi_send_checkbox = fields.Boolean(
+        string='Send E-Factura to SPV',
+        compute='_compute_l10n_ro_edi_send_checkbox', store=True, readonly=False,
+        help='Send the CIUS-RO XML to the Romanian Government via the ANAF platform')
+    l10n_ro_edi_send_warning_message = fields.Text(compute='_compute_l10n_ro_edi_send_warning_message')
+
+    def _get_wizard_values(self):
+        # EXTENDS 'account'
+        values = super()._get_wizard_values()
+        values['l10n_ro_edi_send'] = self.l10n_ro_edi_send_checkbox
+        return values
+
+    @api.depends('move_ids.ubl_cii_xml_id', 'enable_ubl_cii_xml')
+    def _compute_l10n_ro_edi_send_enable(self):
+        for wizard in self:
+            wizard.l10n_ro_edi_send_enable = wizard.enable_ubl_cii_xml or any(
+                m.ubl_cii_xml_id and m.l10n_ro_edi_state != 'invoice_sent' for m in wizard.move_ids)
```

Comment:
I think `m.l10n_ro_edi_state != 'invoice_sent'` is wrong here because it's passing with a BIS-3 without any value on `l10n_ro_edi_state`. That's why `!=` could be dangerous :p